problems/1.types+operators.js

```diff
@@ -1,21 +1,46 @@
 /** Takes in an value and returns if it is a number
-* @param {any} val - The input value
-* @returns {boolean} - Is val a number
-*/
-function isNumber() {
+ * @param {any} val - The input value
+ * @returns {boolean} - Is val a number
+ */
+function isNumber(a, b) {
+  let type1 = typeof a;   
+  let type2 = typeof b;
+
+  return type1 === "number"; "string"; "NaN" && type2 === "number"; "string"; "NaN";
 
 }
 
+
+    
+        
+
+    
+    
+
 /** Takes in a value and returns whether it is truthy
-* @param {any} val - The input value
-* @returns {boolean} - Is val truthy
-*/
+ * @param {any} val - The input value
+ * @returns {boolean} - Is val truthy
+ */
 function isTruthy() {
+   let i = 0;
+   while (i <= 0) {
```

Comment:
No need for loops here. or i

Reply:
You have to check for truthyness of a given value (parameter). Remember the only falsey values in JS are 0, "",  null, undefined, NaN, false.

Reply:
One thing in this problem that i found today is me failing to read the error message and de bugging rather than just changing the code.